adr/2023-01-06-achievements.md

```diff
@@ -0,0 +1,23 @@
+# Achievements
+
+We need to know whether achievements should exist for every course or course independent.
+
+## Possible Solutions
+
+1. Achievements exists for every course and progress is only tracked within that course
+
+2. Achievements are course independent and progress is tracked across all courses
+
+## Chosen Solution
+
+2. Achievements are course independent and progress is tracked across all courses
+
+## Pro Chosen Solution
+
+- No repetetive unlocking of the same achievements in different courses
+- More options for long term achievements (complete five courses, ...)
```

Comment:
```suggestion
- More options for long term achievements (complete five courses, ...)
- Easier to implement as it can be hardcoded instead of dynamically inserted
```